docs/content/drafts/PageLayout.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: PageLayout
+status: Draft
+description: TODO
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+## Examples
+
+### Default
+
+```jsx
+<PageLayout>
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### With dividers
+
+```jsx
+<PageLayout>
+  <PageLayout.Header showDivider>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane showDivider>Pane</PageLayout.Pane>
+  <PageLayout.Footer showDivider>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### Condensed
+
+```jsx
+<PageLayout columnGap="condensed" rowGap="condensed">
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+## Props
+
+### PageLayout
+
+<!-- TODO: Resoponsive variants -->
+
+<PropsTable>
+  <PropsTableRow
+    name="containerWidth"
+    type={`| 'full'
+| 'medium'
+| 'large'
+| 'xlarge'`}
```

Comment:
@vdepizzol Do you prefer `md/lg/xl` or `medium/large/xlarge`? 

I plan to start a larger discussion about size naming across the Primer ecosystem, but for now, I'm happy to move with whichever names you prefer to avoid blocking this component.

Reply:
@colebemis whatever feels more at home with the current PRC nomenclature works for me. I've used `md/lg/xl` to match with the `container-**` PCSS classes.